File: digiarchive/src/main/java/com/archive/dto/DocumentDto.java
package com.archive.dto;

import lombok.*;

import java.util.Date;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@ToString
public class DocumentDto {

    private String document_id;
    private String conserv_unit_id;
    private Date archiving_reference_date;
    private String frozen_label;
    private boolean hold_status;
    private boolean frozen;
    private String context_id;
    private String classification_nature_id;
    private String classification_nature_label;
    private int classification_nature_code;
    private int duration;
    private String id_event;
    private String event_type;
    private Date event_date;

}

File: archivage-angular/src/app/models/documentDto.ts


export class DocumentDto {

  file_name: string;
  archive_format: string;
  encodedDoc: string;
  contextDtoIn: ContextDtoIn = new ContextDtoIn();

}

class ContextDtoIn {

  conserv_unit_id: number;
  mine_type: string;
  archiving_reference_date: Date;
  final_business_processing_date: Date;
  frozen_label: string;
  hold_status: boolean;
  frozen: boolean;
  final_hold_date: Date;
  deletion_date: Date;
  classification_natureId: number;
  eventType: number = 0;
  customerId: number;
  contractId: number;
  accountId: number;

}
